src/components/App.tsx

```diff
@@ -422,21 +417,13 @@ export class App extends React.Component<AppProps, AppState> {
     const gistURI = await Service.exportProjectToGist(this.project, this.state.fiddle);
     this.logLn("Project Gist CREATED ");
     if (gistURI) {
-      this.setState({
-        toast:  gistURI
-      });
+      this.toastInstance.showToast(<span>"Gist Created!" <a href={gistURI} target="_blank" className="toast-span">Open in new tab.</a></span>);
```

Comment:
This is okay for now, but I think it's a React anti-pattern. I'm not a React expert by any means, but I think elements should only be created in render functions.